ddtrace/profiling/collector/__init__.py

```diff
@@ -77,5 +77,5 @@ def _create_capture_sampler(collector):
 
 @attr.s
 class CaptureSamplerCollector(Collector):
-    capture_pct = attr.ib(factory=attr_utils.from_env("DD_PROFILING_CAPTURE_PCT", 1.0, float))
+    capture_pct = attr.ib(factory=attr_utils.from_env("DD_PROFILING_CAPTURE_PCT", 1.0, float))  # type: ignore[arg-type]
```

Comment:
Let's be sure to report this upstream to mypy, can we also outline it in the PR description?

Reply:
envier FTW! 🙂